Tests/GitHubLabels.tests.ps1

```diff
@@ -271,6 +271,61 @@ if ($script:accessTokenConfigured)
 
         $null = Remove-GitHubRepository -OwnerName $script:ownerName -RepositoryName $repositoryName
     }
+
+    Describe 'Adding and removing labels to an issue'{
```

Comment:
It doesn't feel like `Describe`, `Context`, and `It` are being used correctly here.  My understanding is that `Describe` is intended to be scoped for a single function.  So `Describe 'Add-GitHubLabel'`...  `It 'does this thing...'` and `It 'does this other thing...'`.  Optionally you can add in `Context` for when the behavior is dependent on other state, e.g. `Context: 'Label already exists'`.  You can _use_ these constructs as it is here, but it doesn't seem to match the intent of how they should be used.